# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../powertop/no_unicode.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Inspired by Gentoo, needs further revamping
--- ./Makefile.orig	2007-08-20 01:43:08.000000000 +0800
+++ ./Makefile	2007-09-10 00:45:06.000000000 +0800
@@ -20,7 +20,7 @@
 	
 
 powertop: $(OBJS) Makefile powertop.h
-	$(CC) ${CFLAGS}  $(OBJS) -lncursesw -o powertop
+	$(CC) ${LDFLAGS}  $(OBJS) -lncurses -o powertop
 	@(cd po/ && $(MAKE))
 
 powertop.1.gz: powertop.1
